app.py

```diff
@@ -220,4 +282,5 @@ def tootfeed(query_feed):
 
 
 if __name__ == "__main__":
-    app.run()
+    app.run(use_reloader=True)
```

Comment:
This was helpful during dev.  I haven't deployed it, but I suspect it won't have any effect under gunicorn or whatever.